cmd/deploy/local.go

```diff
@@ -224,6 +226,11 @@ func (ld *localDeployer) runDeploySection(ctx context.Context, opts *Options) er
 		oktetoLog.SetStage("")
 	}
 
+	err = ld.ConfigMapHandler.updateEnvs(ctx, opts.Name, opts.Manifest.Namespace, opts.Variables)
```

Comment:
Are you using the right variables? opts.Variables are the ones defined by the user, or am I wrong?

Reply:
Oh I didn't see that we were adding all the variables from the okteto env file to opts.variables